.travis.yml

```diff
@@ -6,17 +6,20 @@ services:
 # python version
 python:
   - "3.6.0"
+
 # command to install dependencies
 install:
-  - pip install -r requirements.txt 
+  - pip install -r requirements.txt
+  - export SECRET_KEY="this-is-hard-to-break"
```

Comment:
I would suggest you store the SECRET_KEY as an environment variable in the travis settings as there is an option for that.

Reply:
Thanks for the feedback. It has been addressed